matrix-sdk/src/main/java/org/matrix/androidsdk/adapters/MessageRow.java

```diff
@@ -35,8 +54,12 @@
     public MessageRow(Event event, RoomState roomState) {
         mEvent = event;
         mRoomState = roomState;
-    }
 
+        mUserDisplayName = (null == roomState) ? event.getSender() : roomState.getMemberName(event.getSender());
+        mRoomCreateContent = (null == roomState) ? null : roomState.getRoomCreateContent();
+        mSender = roomState.getMember(event.getSender());
```

Comment:
1. What happen if roomState is null?
2. A deep copy of the room member should be useful here. Then mUserDisplayName should be useless

Reply:
Dealing with point 1 makes point 2 non applicable